config.cfg

```diff
@@ -43,6 +43,10 @@ wireguard_PersistentKeepalive: 0
 wireguard_network_ipv4: 10.19.49.0/24
 wireguard_network_ipv6: fd9d:bc11:4021::/48
 
+# Used to segregate the incoming traffic to a separate IP
+# Available for the following cloud providers: DigitalOcean
+static_ip: true
```

Comment:
@davidemyers How do you find `alternative_ingress_ip` suitable for this?

Reply:
Yes, thanks.